### ISSUE TITLE
Tiddler not removed

### ISSUE THREAD

**Issue by adfaure** [Jermolene/TiddlyWiki5#2815]
Hi, whenever I want to delete a tiddler it re-appears after one refresh.
The delete button seems to have no effect. 

I am doing it wrong ? 
I did : more actions button -> delete.

I am running with 
```bash
$ tiddlywiki  --version
> 5.1.13

$ node -v
> v7.8.0
```

Thanks.

**Comment by pmario** [comment]
There isn't enough info. Can you post the exact commands, that you use, so we can replicate the situation. 
eg: 

```
mkdir tmp
cd tmp
tiddlywiki ./test --init server
tiddlywiki ./test --server
```

open `http://localhost:8080` in your browser and try again

The command line produces a log ... 

we need that log. 
and the `tiddlywiki.info` from the `./test` directory.

**Comment by twMat** [comment]
Could it be that the tiddlers you're trying to delete are so called shadow tiddlers, i.e tiddlers that have the prefix `$:/` and typically appear in **bold** writing, like this **$:/foobar** ?

Such tiddlers cannot be deleted because they are part of plugins so one has to delete the whole plugin.

**Comment by adfaure** [comment]
Hi ! Many thanks for your help.
 @twMat: I try to delete a tiddler that I created from the wiki, so it is not a shadow tiddler.
@pmario : So I did what you asked.
Here are the output lines when I delete the tiddler:
```
syncer-server: Dispatching 'save' task: $:/StoryList
FileSystem: Saved file /home/adfaure/phdwiki/tiddlers/$__StoryList.tid
syncer-server: Dispatching 'save' task: $:/StoryList
FileSystem: Saved file /home/adfaure/phdwiki/tiddlers/$__StoryList.tid
```
i also created a  gist for the [`tiddlywiki.info`](https://gist.github.com/adfaure/777f99caa2d21b54f992ee1da95ab221) 

I hope this is enough.

**Comment by pmario** [comment]
I did the following: ... 

 - start server
 - open: http://localhost:8080
 - create New Tiddler
 - save New Tiddler
 - edit New Tiddler
 - delete New Tiddler

which created this: 

```
$ tiddlywiki ./test --server
Serving on 127.0.0.1:8080
(press ctrl-C to exit)
syncer-server: Dispatching 'save' task: $:/StoryList
FileSystem: Saved file D:\git\tmp\test\tiddlers\$__StoryList.tid
syncer-server: Dispatching 'save' task: $:/StoryList
FileSystem: Saved file D:\git\tmp\test\tiddlers\$__StoryList.tid
syncer-server: Dispatching 'save' task: Draft of 'New Tiddler'
FileSystem: Saved file D:\git\tmp\test\tiddlers\Draft_of_'New_Tiddler'.tid
syncer-server: Dispatching 'delete' task: Draft of 'New Tiddler'
FileSystem: Deleted file D:\git\tmp\test\tiddlers\Draft_of_'New_Tiddler'.tid
syncer-server: Dispatching 'save' task: $:/StoryList
FileSystem: Saved file D:\git\tmp\test\tiddlers\$__StoryList.tid
syncer-server: Dispatching 'save' task: New Tiddler
FileSystem: Saved file D:\git\tmp\test\tiddlers\New_Tiddler.tid
syncer-server: Dispatching 'save' task: $:/StoryList
FileSystem: Saved file D:\git\tmp\test\tiddlers\$__StoryList.tid
syncer-server: Dispatching 'save' task: $:/StoryList
FileSystem: Saved file D:\git\tmp\test\tiddlers\$__StoryList.tid
syncer-server: Dispatching 'save' task: Draft of 'New Tiddler'
FileSystem: Saved file D:\git\tmp\test\tiddlers\Draft_of_'New_Tiddler'.tid
syncer-server: Dispatching 'delete' task: New Tiddler
FileSystem: Deleted file D:\git\tmp\test\tiddlers\New_Tiddler.tid
syncer-server: Dispatching 'delete' task: Draft of 'New Tiddler'
FileSystem: Deleted file D:\git\tmp\test\tiddlers\Draft_of_'New_Tiddler'.tid
syncer-server: Dispatching 'save' task: $:/StoryList
FileSystem: Saved file D:\git\tmp\test\tiddlers\$__StoryList.tid
```

That's how it should look like.

**Comment by pmario** [comment]
your tiddlywiki.info looks ok

**Comment by pmario** [comment]
which OS do you use?

**Comment by pmario** [comment]
I did test it with win10-git-bash and ubuntu 16.10 ... works as intended.

**Comment by adfaure** [comment]
```
lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 8.6 (jessie)
Release:        8.6
Codename:       jessie
```

**Comment by pmario** [comment]
the password option doesn't make sense here! 

see: http://tiddlywiki.com/#PasswordCommand:PasswordCommand%20ServerCommand

... but I still can't replicate the issue, if I use it like you did. ...

... I'm sorry, but I have no clue atm. @Jermolene

**Comment by adfaure** [comment]
Ah? Maybe, but still the wiki ask for the password when I connect to it ... 

Okay, @pmario thanks for investigating, I will continue to investigate on my side to. Besides I could not find any tuto/doc for creating plugins. Do you know where I could find some ?

**Comment by twMat** [comment]
>Besides I could not find any tuto/doc for creating plugins. Do you know where I could find some ?

There are no docs, at least not that I know of, but you could make a search on the boards for Tinka by @Drakor . Great thingy and it was recently updated.

**Comment by Jermolene** [comment]
Hi @adfaure what version of TiddlyWiki are you using? It's worth trying with the latest prerelease: http://tiddlywiki.com/#Installing%20TiddlyWiki%20Prerelease%20on%20Node.js

**Comment by Jermolene** [comment]
> Besides I could not find any tuto/doc for creating plugins

See http://tiddlywiki.com/dev/#Developing%20plugins%20using%20Node.js%20and%20GitHub:%5B%5BDeveloping%20plugins%20using%20Node.js%20and%20GitHub%5D%5D%20%5B%5BHow%20to%20create%20plugins%20in%20the%20browser%5D%5D

**Comment by adfaure** [comment]
Hello, thanks for your answer.
I just tried to update the command I use to run the server, and it finally works.

For the record I used:
``node /home/adfaure/.nvm/versions/node/v7.8.0/bin/tiddlywiki phdwiki/ --server --password phd``

and as @pmario said the command is wrong, so I updated it to :

```node /home/adfaure/.nvm/versions/node/v7.8.0/bin/tiddlywiki phdwiki/ --server 8080 $:/core/save/all text/plain text/html adfaure phd```

So it as all my bad. 
Thanks for the help.

@Jermolene  Thank you for the link to develop plugins.
